src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selected = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+  const isSelected = tab => tab.id === selected.id;
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={(cn({ 'is-active': isSelected(tab) }))}
+              data-cy="Tab"
+              key={tab.id}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => {
+                  if (!isSelected(tab)) {
+                    onTabSelected(tab);
+                  }
+                }}
```

Comment:
To improve readability and maintainability, it is better to move this to a separate handler